### ISSUE TITLE
Version v5.3.0 was not published on npm

### ISSUE THREAD

**Issue by trivikr** [saebekassebil/microphone-stream#49]
## Describe the bug
Version v5.3.0 was not published on npm https://www.npmjs.com/package/microphone-stream
GitHub Release was successful though https://github.com/microphone-stream/microphone-stream/releases/tag/v5.3.0

#### Expected behavior
New version should be published on npm when GitHub release happens.

**Comment by trivikr** [comment]
The npm version wasn't published as Travis CI didn't run.
Related issue tracking move to GitHub Actions: https://github.com/microphone-stream/microphone-stream/issues/43

**Comment by nfriedly** [comment]
Oh, this is because we moved it to the microphone-stream org. I think I can fix it.

**Comment by nfriedly** [comment]
Travis is updated to know about the new org now, it didn't have permission to access it before. It just ran a test build and then published v5.3.1 to npm.

(I know travis is dying and we should move to github actions, but I don't think I have the time to get a headless browser working on github actions tonight.)

**Comment by trivikr** [comment]
Verified that `v5.3.1` was published on npm.

```console
$ npm view microphone-stream versions
[
  '1.0.0', '1.0.1', '2.0.0',
  '2.0.1', '2.0.2', '3.0.0',
  '3.0.1', '3.0.2', '3.0.4',
  '3.0.5', '3.0.6', '3.0.7',
  '3.0.8', '4.0.1', '4.1.2',
  '4.1.3', '5.0.1', '5.1.0',
  '5.2.0', '5.3.1'
]
```
